src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -204,5 +204,12 @@
             </property>
             <deprecated>The service "%service_id%" is internal and deprecated since Symfony 3.3 and will be removed in Symfony 4.0</deprecated>
         </service>
+
+        <!-- Commands -->
+        <service id="form.command.debug" class="Symfony\Component\Form\Command\FormDebugCommand">
```

Comment:
console.xml is only registered if `Console\Application` class exists.

With #23624 i kept every service in console.xml, and remove the definition if the component is disabled. But the definition is only available with console, so it works for both.